mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
I can't find this comment on github, but core seems like an awful idea.
Why would anyone ever want to see "core" of, say, an image card.

And why do you think they would never want a uri_link for a non-URI card?
 seems highly useful.

On Thu, Jan 1, 2015 at 1:04 PM, Gerry Gleason <notifications@github.com>
wrote:

> In mod/05_standard/set/type/uri.rb
> <https://github.com/wagn/wagn/pull/281#discussion-diff-22401972>:
>
> > @@ -0,0 +1,11 @@
> > +format do
> > +  include Phrase::Format
> > +
> > +  view :core do |args|
> > +    build_link _render_raw(args), render_title(args) || card.name
> > +  end
> > +
> > +  view :uri do |args|
> > +    build_link _render_raw(args)
>
> Ok, url_link makes more sense. You don't think this view is at all related
> to the existing url view?
>
> What should that view give on other types? Seems illogical for it to be a
> link at all, which is why I think it should come back as core or raw view
> (core seems right, but I'm not sure it matters). We just want to see
> something when this view is used on other types, otherwise debugging is
> harder.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/281/files#r22401972>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater